qubes-builder-docker/entrypoint.sh

```diff
@@ -14,6 +14,12 @@ fail() {
 [ -n "$BASE_COMMIT" ] || fail "\$BASE_COMMIT is empty"
 [ -n "$SPEC_PATTERN" ] || fail "\$SPEC_PATTERN is empty"
 
+echo "Before: $SPEC_FILENAME"
```

Comment:
Did you intend to keep these two output lines?  In this form they will likely puzzle anyone who'll be reading build logs.

Reply:
No, they were just for debugging, thanks.